src/styles/colors.scss

```diff
@@ -12,3 +12,4 @@ $swirl: #d7cec4;
 $vanilla: #d0c1a8;
 $double-spanish-white: #cfc2a8;
 $nobel: #989898;
+$preview-panel: #e8ecf4;
```

Comment:
One last minor request: We've been using color descriptions rather than the use of the color. You can pick, but this seems like a `$greyblue` to me.

Reply:
Changed.